rackspace/networking/v2/ports/delegate.go

```diff
@@ -24,6 +24,9 @@ func Get(c *gophercloud.ServiceClient, networkID string) os.GetResult {
 
 // Create accepts a CreateOpts struct and creates a new network using the values
 // provided. You must remember to provide a NetworkID value.
+//
+// NOTE: Currently the SecurityGroup option is not implemented to work with
+// Rackspace.
```

Comment:
I wish we had a way to log a warning if users tried to use it, for example. Logging from a library seems like a bad idea, though.